guides/source/testing.md

```diff
@@ -931,7 +931,7 @@ publishing draft articles, etc.
 Integration Testing
 -------------------
 
-Integration tests are used to test how various parts of your application interact. They are generally used to test important workflows within our application.
+Integration tests are used to test how various parts of your application interact. They are generally used to test important workflows within your application.
```

Comment:
This guide seems switch back and forth between "your" and "our" (and "you" and "we").  Perhaps that should be addressed in another PR, but for now I think we can just maintain consistency within each section.  Since the sentences following this one use "our", how about:

```suggestion
Integration tests are used to test how various parts of our application interact. They are generally used to test important workflows within our application.
```